content/docs/command-reference/config.md

```diff
@@ -249,10 +249,16 @@ experiments or projects use a similar structure.
 
 ### plots
 
+- `plots.auto_open` - if `true`, DVC will automatically open the HTML file
+  generated by `dvc plots` commands in a browser. `false` by default
+
 - `plots.html_template` - sets a
   [custom HTML template](/doc/command-reference/plots#html-templates) for
   `dvc plots`. Accepts a path relative to the `.dvc/` folder.
 
+- `plots.out_dir` - changes the default value for `dvc plots show --outs` and
+  `dvc plots diff --out`. The original default value is `dvc_plots`.
```

Comment:
Looks like we also need to update the default value info in https://dvc.org/doc/command-reference/plots/show#-o and https://dvc.org/doc/command-reference/plots/diff#-o ?